react-backend/client/src/shared/pages/Home.js

```diff
@@ -26,14 +27,14 @@ const FlexItem = styled.div`
   flex: ${props => props.flex || '1'};
 `;
 
-const ButtonLeft = styled(PrimaryButtonNoFill)`
-  width: 30vh;
-  height: 25vh;
+const ButtonLeft = styled(DotButton)`
```

Comment:
Same here, should it be extend?

Reply:
This causes a runtime error, so wrapping is the correct way.